index.html

```diff
@@ -834,34 +844,35 @@ <h4>Detecting when the next media frame will be rendered</h4>
       <h2>Recommendations</h2>
       <p>
         This section describes recommendations from the Media &amp; Entertainment
-        Interest Group for the development of a generic <a>media timed event</a> API,
-        and associated synchronization considerations.
+        Interest Group for the development of a generic <a>timed metadata</a>
+        API, and associated synchronization considerations.
       </p>
       <section>
-        <h3>Subscribing to event streams</h3>
+        <h3>Subscribing to receive timed metadata cues</h3>
         <p>
           The API should allow web applications to subscribe to receive specific
-          event streams by event type. For example, to support MPEG-DASH
+          schemas of <a>timed metadata</a> cue. For example, to support MPEG-DASH
           <code>emsg</code> and MPD events, the API should allow subscription by
           <code>id</code> and (optional) <code>value</code>. This is to make
-          receiving events opt-in from the application point of view. The user
-          agent should deliver only those events to a web application for which
-          the application has subscribed. The API should also allow web
-          applications to unsubscribe from specific event streams by event type.
+          receiving the cues opt-in from the application's point of view. The
```

Comment:
s/the cues/cues grouped by type

Reply:
Resolved in https://github.com/w3c/me-media-timed-events/commit/a33e48b08e37a4d445898b5ef6b3c1f73e3b5728